doc/Adhoc_Testing.md

```diff
@@ -0,0 +1,131 @@
+
+The default generated files, which can be overridden via the appropriate
+options, are:
+
+  * `/tmp/test_collected_results.yaml` for simulated collected results.
+
+  * `/tmp/test_linux_systems.yaml` for simulated systems details.
+
+For example to generate a simulated collected results, and associated systems
+details for a mixture of 2 vCenters, each with 10 ESX hosts running 50 SLE VMs,
+and 20 Libvirt hosts each running 50 SLE VMs, you could use a command like the
+following:
+
+```
+$ .tox/dev/bin/python3 bin/gen_large_datasets -V 2 -E 5 -e 100 -L 20 -l 50 -S mixed2k
+```
+
+
+## `bulk_create`
+
+```
+usage: bulk_create [-h] [--systems_per_request SYSTEMS_PER_REQUEST] [--rmt_uuid RMT_UUID] [--rmt_name RMT_NAME] [-a API_BASE_URL]
+                   [-u USERNAME] [-p PASSWORD] [-s SYSTEMS] [-c]
+
+Bulk create systems SCC VirtualizationHosts API testing
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --systems_per_request SYSTEMS_PER_REQUEST
+                        The max number of systems to upload per request to the SCC. (Default: 200)
+  --rmt_uuid RMT_UUID   The RMT UUID to report to the SCC. (Default: ecf431f8-7faa-48a4-a3f1-6d3abb199086)
+  --rmt_name RMT_NAME   The RMT name to report to the SCC. (Default: rmt.test.example.com)
+  -a API_BASE_URL, --api_base_url API_BASE_URL
+                        The SCC API base URL. (Default: http://localhost:3000)
+  -u USERNAME, --username USERNAME
+                        The SCC mirroring credentials username to use for bulk creation. Can be specified via SCC_USERNAME env var.
+  -p PASSWORD, --password PASSWORD
+                        The SCC mirroring credentials password to use for bulk creation. Can be specified via SCC_PASSWORD env var.
+  -s SYSTEMS, --systems SYSTEMS
+                        The input file containing the details of the Libvirt host and VM systems to bulk create. (Default:
+                        /tmp/test_linux_systems.yaml)
+  -c, --check_creds     Verify that the supplied SCC mirroring credentials are valid.
+```
+
+The `bulk_create` tool takes as arguments:
+
+  * the SCC credentials to use to upload the simulated systems, which can be
+    alternatively specified via the SCC_USERNAME and SCC_PASSWORD environment
+    variables.
+
+  * the SCC API URL to use.
+
+  * the simulated systems details file generated by the `gen_large_datasets`
+    tool.
+
+  * optional simulated RMT identification details.
+
+  * optional number of systems to upload per request
+
+The specified SCC credentials can be validated using the `--check_creds`
+option as well.
+
+For example to create the simulated systems associated with the above
+generated `mixed2k` dataset in a SCC development environment running on
+localhost:5000, you could use a command like the following:
+
+```
+$ bin/bulk_create -s /tmp/test_linux_systems.yaml -a http://localhost:5000 -u ${SCC_ACCOUNT} -p ${SCC_PASSWORD}
```

Comment:
port is 3000

Reply:
This was intended as an example of showing how to specify a different port than the default... The default value for the -a option is http://localhost:3000, but the help output above.